src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -187,9 +941,8 @@ void emberAfThermostatClusterServerInitCallback(chip::EndpointId endpoint)
     // or should this just be the responsibility of the thermostat application?
 }
 
-Protocols::InteractionModel::Status
-MatterThermostatClusterServerPreAttributeChangedCallback(const app::ConcreteAttributePath & attributePath,
-                                                         EmberAfAttributeType attributeType, uint16_t size, uint8_t * value)
+imcode MatterThermostatClusterServerPreAttributeChangedCallback(const app::ConcreteAttributePath & attributePath,
```

Comment:
because thermostat has - using imcode = Protocols::InteractionModel::Status; I was changing everywhere i use Status. i will revert this line. don't want to rename imcode cuz that will add unnecessary changes

Reply:
reverted